scripts/packages.js

```diff
@@ -1,6 +1,20 @@
 module.exports = [
   ['core', {}],
-
+  ['rxjs', {
+    name: '🔥 Rxjs',
```

Comment:
Is there any emoji that could sort of representing Rxjs?

Reply:
idk xD maybe ✨ or 🔬 since everything is about observables... xd

Reply:
Haha, I guess it's better to just remove the emoji part along with the firebase as well 😂